.azure-devops/analysis.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+# Required for schedule only trigger
+trigger: none
+pr: none
+
+# Run weekly at midnight (Pacific).
+schedules:
+- cron: "0 8 * * 1"
```

Comment:
We should think about varying the schedule for this analysis pipeline (adjust day or hour) to balance the agent load across other analysis demands